hacking/test-module

```diff
@@ -209,6 +211,7 @@ def runtest(modfile, argspath, modname, module_style, interpreters):
 
     cmd = subprocess.Popen(invoke, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
     (out, err) = cmd.communicate()
+    out, err = to_text(out, errors='surrogate_or_strict'), to_text(err)
```

Comment:
This one it's okay to use the default of 'replace' for errors since the only thing we do with it is print it to the screen.  So you can revert it to what you had before:
``` python
out, err = to_text(out), to_text(err)
```